src/components/SecondaryPanes/Breakpoints.js

```diff
@@ -248,11 +248,16 @@ class Breakpoints extends PureComponent {
       click: () => setBreakpointCondition(breakpoint.location)
     };
 
-    const editCondition = {
+    const addOrEditCondition = {
       id: "node-menu-edit-condition",
-      label: editConditionLabel,
-      accesskey: editConditionKey,
-      click: () => toggleConditionalBreakpointPanel(breakpoint.location.line)
+      label: addOrEditConditionLabel,
+      accesskey: addOrEditConditionKey,
```

Comment:
I think we should have two separate labels "Add Condition", "Edit Condition" that we toggle based on whether there is a condition. We do the same thing when you right click on the gutter.